libs/domain/picking/src/services/adapter/picking-list-default.adapter.ts

```diff
@@ -201,6 +201,10 @@ export class PickingListDefaultAdapter implements PickingListAdapter {
       createdAt: new Date(data.createdAt),
       updatedAt: new Date(data.updatedAt),
       requestedDeliveryDate: new Date(requestedDeliveryDate),
+      itemsCount: parsedPickingItems.reduce(
+        (count, item) => count + item.quantity,
+        0
+      ),
```

Comment:
Please remove duplicated `itemsCount` parsing logic from the `PickingListOnlineDefaultAdapter` and `PickingListOffline`.
https://github.com/spryker/oryx/blob/a61711e1a09df11c5210b2888d7f3469faadeb40/libs/domain/picking/offline/services/adapter/picking-list-online-default.adapter.ts#L86-L89
https://github.com/spryker/oryx/blob/a61711e1a09df11c5210b2888d7f3469faadeb40/libs/domain/picking/offline/entities/picking-list.entity.ts#L74